### PR TITLE
Move component definitions to `parameters.components`

### PR DESCRIPTION
**Pull request by simu**
This PR changes the Commodore defaults to be ready for https://github.com/projectsyn/commodore/pull/284

### CODE REVIEW

**Review by srueg — APPROVED** [review]
We could also leave them in `commodore.yml` and move them under `parameters:`, right?

**Review by corvus-ch — APPROVED** [review]
LGTM but please do not merge unless all the related documenation is up to date and use a compatible commodore version.

**Comment by simu** [comment]
> We could also leave them in `commodore.yml` and move them under `parameters:`, right?

No, that way they'd have precedence over whatever's configured in the dynamic hierarchy. That's why we have `params.yml` in the first place.

**Pull request merged**
